src/index.js

```diff
@@ -42,12 +42,19 @@ export default function Loadable<Props: {}, Err: Error>(opts: Options) {
   let outsidePromise = null;
   let outsideError = null;
 
-  if (serverSideRequirePath) {
+  if (!isWebpack && serverSideRequirePath) {
     outsideComponent = tryRequire(serverSideRequirePath);
   }
 
   if (isWebpack && webpackRequireWeakId) {
-    outsideComponent = tryRequire(webpackRequireWeakId());
+    let weakId = webpackRequireWeakId();
+    if (__webpack_modules__[weakId]) {
```

Comment:
We could do that, though we're pretty much guaranteed that `__webpack_modules__` will be an object if `isWebpack` is true. I could also change `isWebpack` to be:

```javascript
let isWebpack = typeof __webpac_modules__ !== "undefined";
```

Which is what it was before, but it really makes no difference. Happy to do whatever you decide, but I think doing a `typeof` check is unnecessary here.